docker/hoot_external_command_nightly/scripts/RunCommandsAcrossContainers.sh

```diff
@@ -91,13 +102,25 @@ case "$1" in
         OP_URL=${10}
         OUTPUT_GPKG=${11}
         OUTPUT_OSM=${12}
-        run-owt-import ;;
+        OVERPASS_API_HOST=${13}
+        run-owt ;;
```

Comment:
I was thinking to replace `owt` with `overpass` so it didn't refer to a specific overpass instance label.